src/tracker/tracker_worker.h

```diff
@@ -67,7 +67,7 @@ class TrackerWorker {
 
   bool                is_busy_not_scrape() const;
 
-  std::string         tracker_id() const                    { return m_tracker_id; }
+  const auto&         tracker_id() const                    { return m_tracker_id; }
```

Comment:
This is incorrect as it requires a copy due to thread safety.

Reply:
Removed.